src/main/java/net/fabricmc/stitch/commands/tinyv2/TinyClass.java

```diff
@@ -0,0 +1,63 @@
+package net.fabricmc.stitch.commands.tinyv2;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class TinyClass implements Comparable<TinyClass> {
+
+    private final List<String> classNames;
+    private final Collection<TinyMethod> methods;
+    private final Collection<TinyField> fields;
+    private final Collection<String> comments;
+
+    public TinyClass(List<String> classNames, Collection<TinyMethod> methods, Collection<TinyField> fields, Collection<String> comments) {
```

Comment:
Why don't you use the model from mapping parser when you have it?

Reply:
Well, this one came before the model in the mapping parser. I couldn't use this model for loom, because it doesn't have fast access by index, so I made the new model that is backwards compatible with v1. And then I don't see a reason to switch because this model makes it easier to the AST transformations, and the other model makes it easier to do remapping. All in all it just looks like a lot of code because of java, in  kotlin these models would be one 100 line file.